python/cudf/cudf/core/dataframe.py

```diff
@@ -6755,11 +6818,11 @@ def stack(self, level=-1, dropna=True):
         cat    1.0    2.0
         dog    3.0    4.0
         >>> df_multi_level_cols2.stack()
-            height weight
-        cat kg   <NA>    1.0
-            m     2.0   <NA>
-        dog kg   <NA>    3.0
-            m     4.0   <NA>
+               weight height
```

Comment:
This is odd. Did pandas change the order of the outputs from stack?

Reply:
This seems like it could've been a bug before? I would expect this column order from the input ordering.

Reply:
It was a bug, which is now fixed.